### PR TITLE
Update uni-date-picker links and readme

### PR DESCRIPTION
**Pull request by tomalec**

### CODE REVIEW

File: components/index.html [hunk 1]
```diff
@@ -72,6 +72,7 @@ <h2>Table of Contents</h2>
                 <li><a href="README.md">Principles & solution overview</a></li>
                 <li><a href="uni-date-picker/">&lt;uni-date-picker&gt;</a> - date picker UI for native <code>&lt;input type="date"&gt;</code></li>
                 <li><a href="uni-form-item/">&lt;uni-form-item&gt;</a> - form item container for a <code>&lt;label&gt;</code>, a form control of any kind, and an <code>&lt;output&gt;</code> validation message</li>
+                <li><a href="uni-form-item/">&lt;uni-data-table&gt;</a> - a data table with lazy loading and rich declarative cell content</li>
```
Comment (alshakero) [inline components/index.html]: I think the `href` value is incorrect?

Comment (alshakero) [inline components/index.html]: Maybe copy the README definition as is for consistency?

"A custom element providing a data table with lazy loading and rich declarative cell content."

Comment (tomalec) [reply components/index.html]: All of them by definition will be custom elements, isn't starting each entry with "A custom element" too verbose?

Comment (alshakero) [reply components/index.html]: I don't mind any definition you find fit. But it just makes more sense to be consistent in the two places.

**Pull request merged**